conjureup/controllers/newcloud/gui.py

```diff
@@ -26,9 +26,8 @@ def render(self):
         # a user to configure a LXD bridge with suggested network
         # information.
         if cloud_type == 'localhost':
-            lxd = common.is_lxd_ready()
-            if not lxd['ready']:
-                return controllers.use('lxdsetup').render(lxd['msg'])
+            if common.is_lxd_ready():
+                return controllers.use('lxdsetup').render()
```

Comment:
There seems to be a change in the logic here ("not ready" to "ready") but this function actually won't *ever* return anything but `None` (or throw an exception, perhaps), which means `lxdsetup` will never be shown.  This should be inlined to `common.lxd_init()` and perhaps have `try` / `except` logic added here to render the `lxdsetup` view when appropriate (or remove the view / controller entirely).